### ISSUE TITLE
compiled companion

### ISSUE THREAD

**Issue by dklueh79** [EdgeTX/edgetx#108]
It would be great if there could be a compiled companion within the nighty (MacOS, Linux, Win)
Not everyone is able to compile it for the preferred system.

Thx.

**Comment by raphaelcoeffic** [comment]
Certainly, but I think it does not make sense yet, as Companion has not been changed a single bit since the fork. Changes from OTX 2.3 will be merged back here however as possible.

**Comment by pfeerick** [comment]
Compiled companion builds are now available for Windows and Mac, and we will investigate Linux builds soon.
